rasa/shared/core/constants.py

```diff
@@ -80,3 +80,5 @@
 
 USE_TEXT_FOR_FEATURIZATION = "use_text_for_featurization"
 ENTITY_LABEL_SEPARATOR = "#"
+
+POLICIES_THAT_EXTRACT_ENTITIES = {"TEDPolicy"}
```

Comment:
should this be
```suggestion
POLICIES_THAT_EXTRACT_ENTITIES = {f"{TEDPolicy.__class__._name__}"}
```
?
also in TEDPolicy in line `728`?

Reply:
let's keep a string